src/client/src/mockData/extensionModules/mockCoreTSModule.ts

```diff
@@ -1,7 +1,19 @@
 import { EXTENSION_COMMANDS, EXTENSION_MODULES } from "../../utils/constants/commands";
-import { WIZARD_PROJECT_TYPE } from "../../utils/constants/internalNames";
 
-import * as mockData from "./mockData/mockWebPlatformData";
+// const mockData = import("./mockData/mockWebPlatformData");
+import * as mockWebData from "./mockData/mockWebPlatformData";
+import * as mockReactNativeData from "./mockData/mockReactNativePlatformData";
+
+import { PLATFORM } from "../../utils/constants/constants";
+
+const platform = PLATFORM.REACTNATIVE;
```

Comment:
Maybe find a way to make clear what needs changing when debugging and developing on the browser so we clearly know what do we need to change to be able to test WEB or REACT_NATIVE version